server/cli/prepare.py

```diff
@@ -112,7 +117,12 @@ def run_neighbors(adata):
         sc.pp.neighbors(adata)
 
     def run_louvain(adata):
-        sc.tl.louvain(adata)
+        try:
+            sc.tl.louvain(adata)
+        except ModuleNotFoundError:
+            click.echo("\nWarning: louvain module is not installed, no clusters will be calculated. "
```

Comment:
This interrupts the click progress bar so I included a new line so that the warning is on its own line and not after the progress bar